src/css/general.css

```diff
@@ -305,6 +305,10 @@ th {
 This is needed to remove it.
 */
 .katex .accent {
-    background-color:transparent !important;
+    background-color: transparent !important;
+    border-color: transparent !important;
+}
+.v-application .accent {
+    background-color: transparent !important;
```

Comment:
Why not simply:
```css
.katex .accent, .v-application .accent {
    background-color: transparent !important;
    border-color: transparent !important;
}
```

Reply:
Mostly because sooner or later I want to figure out how to not have the `v-application` stuff over-riding things.  I think it means that something is being loaded in a strange order as the css has several layers/loads that I don't quite understand.  I've tried to keep `v-application` blocks separate from others for that reason.  It's ugly, I know ... but will need adjusting everywhere.